tests/parm/nems.configure.cpld_noaero_outwav.IN

```diff
@@ -116,6 +118,7 @@ ALLCOMP_attributes::
       ScalarFieldName = cpl_scalars
       start_type = @[RUNTYPE]
       restart_dir = RESTART/
+      RunTimeLog = true
```

Comment:
Is this the only test we want to add this attribute? This is the waves in the slow loop test. We're not adding waves for most of the scalability testing.

Reply:
I am trying to add the runtime log to the GFSv17 test, cpld_control_gfsv17 test, which is using this nems.configure file.